Demo/obj/Debug/netcoreapp3.0/Razor/Pages/Error.cshtml.g.cs

```diff
@@ -0,0 +1,90 @@
+#pragma checksum "C:\Users\Yas\source\repos\Demo\WebrunsDemo\Pages\Error.cshtml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "1f02565c188bc0cf60de1bc120acef71a3608055"
+// <auto-generated/>
+#pragma warning disable 1591
+[assembly: global::Microsoft.AspNetCore.Razor.Hosting.RazorCompiledItemAttribute(typeof(Demo.Pages.Pages_Error), @"mvc.1.0.razor-page", @"/Pages/Error.cshtml")]
+namespace Demo.Pages
+{
+    #line hidden
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Rendering;
+    using Microsoft.AspNetCore.Mvc.ViewFeatures;
+#nullable restore
+#line 1 "C:\Users\Yas\source\repos\Demo\WebrunsDemo\Pages\_ViewImports.cshtml"
+using Demo;
+
+#line default
+#line hidden
+#nullable disable
+    [global::Microsoft.AspNetCore.Razor.Hosting.RazorSourceChecksumAttribute(@"SHA1", @"1f02565c188bc0cf60de1bc120acef71a3608055", @"/Pages/Error.cshtml")]
+    [global::Microsoft.AspNetCore.Razor.Hosting.RazorSourceChecksumAttribute(@"SHA1", @"d172f22b6a6aa03c697cebcb396c7b0a1b74a5ac", @"/Pages/_ViewImports.cshtml")]
+    public class Pages_Error : global::Microsoft.AspNetCore.Mvc.RazorPages.Page
+    {
+        #pragma warning disable 1998
+        public async override global::System.Threading.Tasks.Task ExecuteAsync()
+        {
+#nullable restore
+#line 3 "C:\Users\Yas\source\repos\Demo\WebrunsDemo\Pages\Error.cshtml"
+  
+    ViewData["Title"] = "Error";
+
+#line default
+#line hidden
+#nullable disable
+            WriteLiteral("\r\n<h1 class=\"text-danger\">Error.</h1>\r\n<h2 class=\"text-danger\">An error occurred while processing your request.</h2>\r\n\r\n");
+#nullable restore
+#line 10 "C:\Users\Yas\source\repos\Demo\WebrunsDemo\Pages\Error.cshtml"
+ if (Model.ShowRequestId)
+{
+
+#line default
+#line hidden
+#nullable disable
+            WriteLiteral("    <p>\r\n        <strong>Request ID:</strong> <code>");
+#nullable restore
+#line 13 "C:\Users\Yas\source\repos\Demo\WebrunsDemo\Pages\Error.cshtml"
```

Comment:
still a lot of wrong named lines like this. Use webRunes or avoid it at all

Reply:
This File does not exist in our future commit